### PR TITLE
Jfb 8 AND JFB-9(last hw)

### PR DESCRIPTION
**Pull request by chingiz-mazhitov**

### CODE REVIEW

File: src/main/java/com/andersen/config/JpaConfig.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Import(DataSourceCfg.class)
+@Configuration
+@ComponentScan(basePackages = {"com.andersen"})
+@EnableTransactionManagement
+@Slf4j
```
Comment (Alexmansar) [inline src/main/java/com/andersen/config/JpaConfig.java]: not needed

**Review by Alexmansar — APPROVED** [review]
@chingiz-mazhitov Hi)) good job) and good idea with docker image) I have 1 small comment) just fix it and merge please)

File: src/main/java/com/andersen/config/JpaConfig.java (hunk 1 above)
Comment (chingiz-mazhitov) [reply src/main/java/com/andersen/config/JpaConfig.java]: without @ComponentScan and @EnableTransactionManagement, I get an error

Comment (Alexmansar) [reply src/main/java/com/andersen/config/JpaConfig.java]: @chingiz-mazhitov this comment was just to Slf4j
so, return Configuration please

**Pull request merged**